src/pages/pass-finalisation/cat-c/pass-finalisation.cat-c.page.html

```diff
@@ -0,0 +1,55 @@
+<ion-header>
+  <ion-navbar hideBackButton>
+    <ion-title>Test debrief - {{pageState.candidateUntitledName$ | async}}</ion-title>
+  </ion-navbar>
+</ion-header>
+
+<ion-content>
+  <div>
+    <form [formGroup]="form" (ngSubmit)="onSubmit()">
+      <ion-grid>
+
+        <finalisation-header [candidateName]="pageState.candidateName$ | async"
+          [candidateDriverNumber]="pageState.candidateDriverNumber$ | async"
+          [outcomeText]="pageState.testOutcomeText$ | async"></finalisation-header>
+
+        <ion-row class="mes-component-row mes-row-separator" id="application-reference-card" align-items-center>
+          <ion-col class="component-label" col-32>
+            <label>Application reference</label>
+          </ion-col>
+          <ion-col padding-left>
+            <span class="mes-data">{{pageState.applicationNumber$ | async }}</span>
+          </ion-col>
+        </ion-row>
+
+        <license-provided [form]="form" (licenseReceived)="provisionalLicenseReceived()"
+          (licenseNotReceived)="provisionalLicenseNotReceived()"> </license-provided>
+
+        <transmission [formGroup]="form" (transmissionChange)="transmissionChanged($event)"></transmission>
+        <warning-banner *ngIf="pageState.transmissionAutomaticRadioChecked$ | async"
```

Comment:
This code contains a bug that affected Cat B and Cat B+E. Easy to fix.

Reply:
Added the feature to cat c as well